src/pages/community/columns/admin-column.component.jsx

```diff
@@ -31,10 +31,10 @@ class AdminColumn extends Component {
   }
 
   getData = async () => {
+    const modifiedData = [];
     const response = await new CommunityAdminsApi(this.props.communityId).getAdmins();
-    response.data.admins.map(v => v.adminId).forEach(async (id) => {
-      const userResponse = await new UsersApi().getUser(id);
-      const modifiedData = this.state.data;
+    response.data.admins.map(async (v) => {
```

Comment:
I think we can use a forEach here, can't we?

Reply:
Yes, we can use it as well. You want me to change it to forEach?

Reply:
Done